webpack/assets/javascripts/react_app/components/MemoryAllocationInput/memoryAllocationInput.scss

```diff
@@ -1,5 +1,9 @@
 @import '../../common/colors.scss';
 
+.container {
+  max-width: none !important;
```

Comment:
instead of using `!important` can you create a more specific rule?

Reply:
This is problematic because the relevant div is located inside the react component and I am not sure I can change the div class.